apps/api/src/app.module.ts

```diff
@@ -4,6 +4,8 @@ import { APP_INTERCEPTOR } from '@nestjs/core';
 import { Type } from '@nestjs/common/interfaces/type.interface';
 import { ForwardReference } from '@nestjs/common/interfaces/modules/forward-reference.interface';
 
+import { EEAuthModule } from '@novu/ee-auth';
```

Comment:
Ahh, this might not work for contributors that won't have access to the ee packages repository, like they won't be able to build it and will get an error when accessing this package.
We need to do the trick with `require` as we have for the `bullmq-pro` package. Something like this, but you have to check:

```
const getExternalModules = () => {
   const modules = [];

   const eeAuth = require('@novu/ee-auth');
   if (eeAuth) {
      modules.push(eeAuth.EEAuthModule);
   }

   return modules;
}

const externalModules: Array<Type | DynamicModule | Promise<DynamicModule> | ForwardReference> = getExternalModules();
```

Reply:
Yeah you are totally right and I am aware of this issue, i am testing currently what would be the cleanest solution for us.